x/project/handler.go

```diff
@@ -93,45 +94,47 @@ func handleMsgCreateProject(ctx sdk.Context, k Keeper, msg MsgCreateProject) sdk
 		),
 	})
 
-	return sdk.Result{Events: ctx.EventManager().Events()}
+	return &sdk.Result{Events: ctx.EventManager().Events()}, nil
 }
 
 func handleMsgUpdateProjectStatus(ctx sdk.Context, k Keeper, bk bank.Keeper,
-	msg MsgUpdateProjectStatus) sdk.Result {
+	msg MsgUpdateProjectStatus) (*sdk.Result, error) {
 
 	projectDoc, err := k.GetProjectDoc(ctx, msg.ProjectDid)
 	if err != nil {
-		return sdk.ErrUnknownRequest("Could not find Project").Result()
+		return nil, sdkerrors.Wrap(did.ErrInvalidDid, "could not find project")
```

Comment:
I think we should have used `sdkerrors.ErrUnknownRequest`.

Reply:
Not sure

Reply:
Let's leave this as is